packages/popover/package.json

```diff
@@ -1,20 +1,33 @@
 {
   "name": "pcln-popover",
-  "version": "2.0.5",
+  "version": "3.0.0",
   "description": "React component for popover",
-  "main": "dist/index.cjs.js",
-  "module": "dist/index.esm.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "sideEffects": false,
```

Comment:
Is the `sideEffects` key still needed now that we removed rollup?

Reply:
I think so because the host app's webpack will look at `sideEffects` while building to determine whether or not the module can be tree shaken.

Reply:
My understanding is yes, for exactly the reason @James300 stated.